src/masternodes/mn_checks.cpp

```diff
@@ -2663,6 +2663,15 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor
             if (!res)
                 return res;
 
+            if (static_cast<int>(height) >= consensus.GreatWorldHeight)
```

Comment:
should it be 
`if (static_cast<int>(height) >= consensus.GreatWorldHeight && subLoan < it->second)`
it means owner can repay entire loan on interestPerBlock == 0?